internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -1083,6 +1127,21 @@ components:
           example: 'latest'
           description: |
             Tag for the created container image
+    PulpOSTreeUploadOptions:
+      type: object
+      additionalProperties: false
+      required:
+        - basepath
+      properties:
+        basepath:
+          type: string
+          description: 'Basepath for distributing the repository'
+        repository:
+          type: string
+          description: 'Repository to import the ostree commit to'
+        server_address:
+          type: string
+          format: uri
```

Comment:
These are the three options we support in weldr.  Do we want them all to be supported through the API?  Maybe server address should be configured instead, or is it useful to have it so IB can take care of it?